src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,13 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        Bulldozer bulldozer = new Bulldozer();
+        bulldozer.initalizeStorage();
```

Comment:
let's create, fill and return a list here, not in Bulldozer class
same for other models